buildpipeline/pipeline.json

```diff
@@ -77,6 +77,23 @@
             "Platform": "arm"
           }
         },
+        {
+          "Name": "Core-Setup-Linux-Arm-BT",
+          "Parameters": {
+            "PB_DockerTag": "ubuntu-16.04-cross-arm64-a3ae44b-20180316023254",
```

Comment:
I've been a bit out of the loop, so forgive my ignorance in this question.

Why do we use `Ubuntu 16` to build?  Will the binaries we produce from this build be able to run on `Ubuntu 14`?  Or is that platform not supported?

Reply:
+ `arm64` is a relatively new platform.  14.04 trusty has ~1yr life yet (4/2019).  arm64 releases will be in preview for a while.  
+ I have done all my development testing on arm64 16.04.  
+ @jashook experience was arm64 14.04 was not working.
+ Seems 16.04 is reasonable for initial releases.  We can try moving to trusty later if needed.